extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/boot/QuarkusPersistenceUnitDescriptor.java

```diff
@@ -28,14 +28,15 @@ public final class QuarkusPersistenceUnitDescriptor implements PersistenceUnitDe
     private final PersistenceUnitTransactionType persistenceUnitTransactionType;
     private final ValidationMode validationMode;
     private final SharedCacheMode sharedCacheMode;
+    private final List<String> managedClassAndPackagesNames;
     private final List<String> managedClassNames;
     private final Properties properties;
     private final boolean reactive;
 
     public QuarkusPersistenceUnitDescriptor(String name,
             QuarkusPersistenceUnitProviderHelper providerHelper,
             PersistenceUnitTransactionType persistenceUnitTransactionType,
-            List<String> managedClassNames,
+            List<String> managedClassAndPackagesNames, List<String> managedClassNames,
             Properties properties, boolean reactive) {
```

Comment:
I believe you did not update calls to this constructor in `HibernateOrmProcessor`, so it likely doesn't compile anymore. Please try to compile locally before pushing :)